src/components/OfferDialog.jsx

```diff
@@ -71,13 +76,16 @@ const OfferDialog = ({
   };
 
   const handleConfirm = () => {
-    if (parseFloat(input) < minAmount) {
+    console.log("Dialog input,min,max:", input.toString(), minAmount.toString(),maxAmount.toString());
+
+    if (input.lt(minAmount.toString())) {
+      console.log('WHAT THE HELL');
       setInputInvalid('Set bigger amount');
-    } else if (parseFloat(input) > maxAmount) {
+    } else if (input.gt(ethers.utils.parseEther(maxAmount.toString()))) {
```

Comment:
I don't think `maxAmount` is a bigNumber. In `ViewListing` it's either user's zoom balance or wmovr balance. Since input is a BigNumber it's prob in wei